File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyHextechDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyHextechDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyHextechDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyHextechDragonKillAnimation());
    }
}

File: test/hud/animations/AllyHextechDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyHextechDragonKillAnimation} from '../../../src/hud/animations/AllyHextechDragonKillAnimation.js';

/**
 * Integration test for ally hextech dragon kill animation on actual hardware.
 */
describe('AllyHextechDragonKillAnimation Integration', () => {
    test('playAllyHextechDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyHextechDragonKillAnimation());
    }, 30000);
});
